ods_ci/tests/Tests/1000__model_serving/1009__model_serving_triton_on_kserve/1009__model_serving_triton_on_kserve.robot

```diff
@@ -240,6 +242,36 @@ Test KERAS Model Inference Via UI(Triton on Kserve)
     ...  AND
     ...  Delete Serving Runtime Template From CLI    displayed_name=triton-keras-rest
 
+Test Python Model Rest Inference Via UI (Triton on Kserve)    # robocop: off=too-long-test-case
+    [Documentation]    Test the deployment of an python model in Kserve using Triton
+    [Tags]    Sanity    RHOAIENG-15374      RunThisTest
```

Comment:
Why `RunThisTest` tag. This is not right

Reply:
will remove it now